api_views/books.py

```diff
@@ -9,11 +10,8 @@
 from app import vuln
```

Comment:
Removing the error_message_helper function without providing an alternative error message formatter could lead to inconsistent error responses